llm-examples/falcon/falcon-7b-instruct-inference.py

```diff
@@ -0,0 +1,349 @@
+# Databricks notebook source
+# MAGIC %md
+# MAGIC # Falcon-7b-instruct Inference on Databricks
+# MAGIC Environment for this notebook:
+# MAGIC - Runtime: 13.1 GPU ML Runtime
+# MAGIC - Instance: `g5.4xlarge` on AWS
+# MAGIC
+# MAGIC GPU instances that have at least 16GB GPU memory would be enough for inference on single input (batch inference requires slightly more memory). On Azure, it is possible to use `Standard_NC6s_v3` or `Standard_NC4as_T4_v3`.
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC ## Install required libraries
+
+# COMMAND ----------
+
+# MAGIC %pip install -q -U torch==2.0.1
+# MAGIC %pip install -q einops==0.6.1
+
+# COMMAND ----------
+
+dbutils.library.restartPython()
```

Comment:
Q: Do we need to restart Python?

Reply:
I guess so since seemed like she installed the python libraries

Reply:
In this case turns out that it's not necessary, just removed it.